apps/python/scevtls.py

```diff
@@ -58,7 +58,8 @@ def readXML(self):
                 if evt.creationInfo().modificationTime() < self._modifiedAfterTime:
                     continue
             except ValueError:
-                continue
+                if evt.creationInfo().creationTime() < self._modifiedAfterTime:
```

Comment:
The `creationInfo()` method might throw an exception so a try-except block is necessary as well.